docs/zkapps/o1js/keccak.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Keccak
+hide_title: true
+description: A comprehensive guide on how to use Keccak-based hashes in o1js.
+keywords:
+  - keccak
+  - sha3
+  - hash
+  - ethereum
+  - mina blockchain
+  - blockchain technology
+  - data structures
+---
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+# Keccak (SHA-3)
+
+Keccak is a flexible cryptographic hash function that provides more security than traditional SHA hash algorithms.
+
+## What is Keccak?
+
+Developed by a team of cryptographers from Belgium, Keccak was the winning submission for the National Institute of Standards and Technology (NIST) SHA-3 competition. Keccak was standardized as SHA-3. Today, Keccak is used in many different applications, including Ethereum and many others.
```

Comment:
```suggestion
Developed by a team of cryptographers from Belgium, Keccak was the winning submission for the National Institute of Standards and Technology (NIST) SHA-3 competition. Keccak was standardized as SHA-3. Keccak is used in many different applications, including Ethereum.
```

Reply:
@Trivo25  This content is so good! 

I've made changes to apply the doc styles and want your input on this suggestion. Our goal is to write timeless documentation, so I suggest we omit "today" 
I think we can also omit "and many others" since we already say Keccak is used in "many different applications"

Wdyt?